packages/ebs-helper/src/helpers.external.ts

```diff
@@ -82,6 +82,20 @@ export function createChatMessageBody(config: EbsCallConfig, extensionVersion: s
 	};
 }
 
+/** @private */
+export function createPubSubGlobalMessageJwtData() {
```

Comment:
Now that there are special cases for the global JWT and message body, you can probably remove the ability to receive `undefined` as broadcaster ID from the other two functions.